countrypicker/src/main/java/com/mukesh/countrypicker/models/Country.java

```diff
@@ -24,16 +28,15 @@ public String getCode() {
 
   public void setCode(String code) {
     this.code = code;
+    if (TextUtils.isEmpty(name)) {
```

Comment:
Thinking I should probably remove this check as if we set to a different country code it will reflect that with name (never happens currently but could)